packages/expo-diag/README.md

```diff
@@ -0,0 +1,48 @@
+<!-- Title -->
+<h1 align="center">
+👋 Welcome to <br><code>expo-diag</code>
```

Comment:
This package will be copy and pasted from issue templates so it doesn't need to have some shorthand name. Use a more descriptive and complete name. The package also doesn't really diagnose any issues, it simply shows the environment info.

Reply:
It's a placeholder, see the bottom todo item:

> - [ ] Coming up with a name

Reply:
'Expo Environment Information Gatherer' (`expo-envinfo`)?

Reply:
`npx expo-env-info` sounds pretty good.